### ISSUE TITLE
Creating a parked subdomain generates incorrect nginx config

### ISSUE THREAD

**Issue by fidelics** [QROkes/webinoly#76]
Steps to reproduce:
1. Create wordpress site example.org
2. Enable ssl for this site
3. Create parked site sub.example.org, pointing to example.org
4. Here is the line from the config that will be generated for nginx:
	_server_name sub.sub.example.org;_

Similar issue has already been discussed in 2019:
https://webinoly.com/support/1438/bugs-in-domain-parking-for-subdomains

Ubuntu 22.04, webinoly v1.17.3

Thank you for attention!

**Comment by QROkes** [comment]
**Thanks a lot for reporting this issue and sorry for the inconvenience!**
Seems like I did the same mistake twice, but now for a different reason.

Please, go to line 1201 in `/opt/webinoly/lib/sites` and change it from this:

```
sudo sed -i "/server_name/c \	$sername" /etc/nginx/sites-available/$domain
sudo sed -i "s/$(escaped_string $parked)/$domain/g"  /etc/nginx/sites-available/$domain
```

To this:

```
sudo sed -i "s/$(escaped_string $parked)/$domain/g"  /etc/nginx/sites-available/$domain
sudo sed -i "/server_name/c \	$sername" /etc/nginx/sites-available/$domain
```

That's it!

**Please, confirm if this is correct!**
Of course, this fix will be included in the next Webinoly release v1.17.4.

Regards.

**Comment by fidelics** [comment]
@QROkes Many thanks for your support!
I can confirm that these edits worked and the subdomain was created correctly.
Moreover, an SSL certificate was even automatically issued for it, which is also great.

Have a nice day and thank you for the project!
